Mathlib/FieldTheory/KrullTopology.lean

```diff
@@ -275,3 +275,15 @@ theorem krullTopology_totallyDisconnected {K L : Type*} [Field K] [Field L] [Alg
 #align krull_topology_totally_disconnected krullTopology_totallyDisconnected
 
 end TotallyDisconnected
+
+lemma IntermediateField.fixingSubgroup_top (K L : Type*) [Field K] [Field L] [Algebra K L] :
+    IntermediateField.fixingSubgroup (⊤ : IntermediateField K L) = ⊥ := by
+  ext
+  simp [IntermediateField.mem_fixingSubgroup_iff, DFunLike.ext_iff]
+
+lemma krullTopology_discreteTopology_of_finiteDimensional (K L : Type) [Field K] [Field L]
```

Comment:
I think this can be an instance, right?

Reply:
Oh yeah -- I hadn't realised DiscreteTopology was a class!